app/views/feed/add.phtml

```diff
@@ -3,7 +3,7 @@
 	/** @var FreshRSS_View $this */
 	if ($this->feed !== null) {
 ?>
-<main class="post">
+<main class="post bookmarklet">
```

Comment:
I added this class to have a little reminder that this `add.phtml` is just the form for the bookmarklet and not the default feed adding form.
I would prefer to rename this file to make it clear.